Makefile

```diff
@@ -136,8 +137,18 @@ lint: .gopathok ${GOLANGCI_LINT}
 	${GOLANGCI_LINT} linters
 	${GOLANGCI_LINT} run
 
-shfmt: ${SHFMT}
-	${SHFMT} -w -i 4 -d $(shell ${SHFMT} -f . | grep -v vendor/)
+shellfiles: ${SHFMT}
+	$(eval SHELLFILES=$(shell ${SHFMT} -f . | grep -v vendor/))
+
+shfmt: shellfiles
+	${SHFMT} -w -i 4 -d ${SHELLFILES}
+
+shellcheck: shellfiles ${SHELLCHECK}
```

Comment:
please add into the help option the new target. Just added: https://github.com/cri-o/cri-o/pull/3480 for shfmt

Reply:
Fixed :+1: